data/pigui/modules/info-view/01-ship-info.lua

```diff
@@ -76,7 +76,7 @@ local function shipStats()
 		{ l.PASSENGER_CABIN_CAPACITY..":",    cabinMaximum },
 		false,
 		{ l.ATMOSPHERIC_SHIELDING..":", atmo_shield and l.YES or l.NO },
-		{ l.ATMO_PRESS_LIMIT..":",      string.format("%d atm", shipDef.atmosphericPressureLimit * atmo_shield_cap) },
+		{ l.ATMO_PRESS_LIMIT..":",      string.format("%d atm", atmo_shield_cap == 0 and shipDef.atmosphericPressureLimit or shipDef.atmosphericPressureLimit * atmo_shield_cap) },
```

Comment:
This can be removed if you change the first line to `math.max(player["atmo_shield_cap"], 1)`.